src/WebJobs.Script.WebHost/Helpers/DelegatingStream.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Helpers
+{
+    public abstract class DelegatingStream : Stream
+    {
+        private Stream _innerStream;
+
+        protected DelegatingStream(Stream innerStream)
```

Comment:
Why is this needed?

Reply:
some random C# quirk with inheriting from Stream class I guess. It's only used in the ZipArchive scenario and I'm removing that since it doesn't make sense in the runtime.